src/test/compile-fail/issue-10536.rs

```diff
@@ -1,31 +0,0 @@
-// Copyright 2014 The Rust Project Developers. See the COPYRIGHT
```

Comment:
The deleted files (such as this one) are moves with changes.

Reply:
why were they changed?

and why in a single commit? :cry:

Reply:
I had to add the error note check, as they were relying only on regex matching the output. I couldn't tell in advance whether a given test would need to be changed before running them as `ui`. I can rewrite the commit history to do so later.